new-bot/src/bot/features/download-video.ts

```
@@ -321,23 +335,24 @@ async function fetchTwitterVideoUrl(twitterUrl: string) {
     }
   } catch (error) {
-    console.error('Error calling Cobalt API:', error);
+    console.error(`Error calling Cobalt API (${useProxy ? 'proxy' : 'direct'}):`, error);
     return { success: false, message: 'Error calling Cobalt API' };
   }
 }
 
-// Function to fetch video URL or download video using Cobalt API
-async function fetchInstagramVideoUrl(instagramUrl: string ) {
+// Function to fetch video URL or download video using Cobalt API with optional proxy
+async function fetchInstagramVideoUrl(instagramUrl: string, useProxy: boolean = false) {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)